Gopkg.toml

```diff
@@ -165,4 +165,7 @@ ignored = ["github.com/Sirupsen/logrus"]
 [[constraint]]
   version = "6.0.0"
   name = "k8s.io/client-go"
- 
\ No newline at end of file
+
+[[constraint]]
```

Comment:
grpc package already includes protobuf library, why add another one?

Reply:
We ran into this bug: https://github.com/golang/protobuf/issues/642. Since GRPC uses this as well, we should only have one version (the latest 1.1).